src/shipit_code_coverage/shipit_code_coverage/codecov.py

```diff
@@ -59,8 +59,8 @@ def download_coverage_artifacts(self, build_task_id):
         test_tasks = [t for t in tasks if taskcluster.is_coverage_task(t)]
         for test_task in test_tasks:
             suite_name = taskcluster.get_suite_name(test_task)
-            # Ignore awsy as it is not actually a suite of tests.
-            if suite_name == 'awsy':
+            # Ignore awsy and talos as they aren't actually suites of tests.
+            if suite_name == 'awsy' or 'talos' in suite_name:
```

Comment:
`awsy` is the exact name, `talos` is a substring.
We could do: `if any(ignore in suite_name for ignore in ['awsy', 'talos'])`

I don't expect this to change often (probably never), so leaving it in the code should be fine.